### DISCUSSION TITLE
Align image to left without float

### DISCUSSION THREAD

**Discussion by gummigroda** [lxndrblz/anatole#396] [Q&A] [answered]
Hi,
Is it possible to align an image to the LEFT with some padding, instead of in the center as it appears now.  All this without "floating".

Doable in markdown or with some kind of CSS tweaking?

**Comment by lxndrblz** [comment]
@gummigroda  Did you try the figure shortcode with option left, as shown here?

https://github.com/lxndrblz/anatole/blob/master/exampleSite/content/english/post/figure-shortcode.md

**Comment by gummigroda** [comment]
Hi,

That works. But I need an option to aligning the image to the left without the underlying text moving to the right of the image. (Which I believed was the "floating" part.)

**Comment by gummigroda** [comment]
@lxndrblz  
It looks like this by default:

![image](https://user-images.githubusercontent.com/15022172/201996546-43b610c0-fa47-4560-b328-de8f1c30b431.png)
 
and I'd like align it left:

![image](https://user-images.githubusercontent.com/15022172/201996864-4961086d-c388-49bd-aa96-7c869ec9bf19.png)

is it possible in any way?

**Reply by lxndrblz** [reply]
@gummigroda Yes, it is possible.

You could add this piece of code to a custom css file to modify the default behavior:
```css
.post__content figure {
    margin: initial;
}
```
The result will look like this:
![image](https://user-images.githubusercontent.com/12200062/202730511-b82b8a84-28b9-46c6-9f35-aba38ddace6c.png)

Hope that helps.

**Comment by gummigroda** [comment]
@lxndrblz 

That works, thanks!
Is there an option to also make this applicable to images "formated" as regular markdown, without the 'figure' ie. `![interface-core1](interface-core1.png)`

**Comment by lxndrblz** [comment, marked as answer]
@gummigroda Ofc, try this:

```css
.post__content img {
    margin-left: inherit;

```

**Comment by gummigroda** [comment]
@lxndrblz 

Ahh, I see the logic now. 
Thanks again and THANKS FOR A GREAT THEME! 🎉

**Comment by lxndrblz** [comment]
No worries.
